website/style.css

```diff
@@ -15,16 +15,31 @@ body {
 }
 
 header {
-  padding: 24px 0px 24px 0px;
+  padding: 48px 0px 32px 0px;
   background-color: white;
 }
 
 .inside-header {
   padding: 0px 32px 0px 32px;
+  display: flex;
+}
+
+.header-content{
+  width: fit-content;
+}
+
+.header-logo {
+  margin-right: 48px;
+}
+
+.ecosystem-title {
+  font-size: 60px;
+  line-height: 70px;
+  font-weight: 300;
 }
 
-.ecosystem-logo {
-  width: 350px;
+.qiskit {
```

Comment:
this is a bit vague. Maybe something like `header-qiskit-text` or `header-qiskit-word`.